content/root/glossary.textile

```diff
@@ -0,0 +1,214 @@
+
+See the "messages documentation":/realtime/messages for further information.
+
+h2(#metachannels). Metachannels
+
+Metachannels are a group of channels that receive metadata related to connections, channels and messages. Examples of metachannels include a log channel that receives certain errors and the lifecycle of channels. All metachannels are in the reserved channel namespace @[meta]:@.
+
+See the "metachannels documentation":/realtime/metachannels for further information.
+
+h2(#mqtt). MQTT
+
+Message Queuing Telemetry Transport (MQTT)) is a lightweight publish-subscribe protocol for minimal code implementations, or low bandwidth networks.
+
+See "Wikipedia":https://en.wikipedia.org/wiki/MQTT for further information.
+
+h2(#mqtt-adapter). MQTT Adapter
+
+The Ably MQTT Adapter can translate between MQTT and the Ably protocol.
+
+See the "MQTT Adapter documentation":https://ably.com/documentation/mqtt for further information.
+
+h2(#presence). Presence
+
+Presence enables clients in a channel to be aware of other presence members in the same channel. Each client has a presence state that is updated as a presence event when it changes, for example when they join or leave the channel.
+
+See the "presence documentation":/realtime/presence for further information.
+
+h2(#pub-sub). Pub/Sub
+
+Publish-subscribe (or pub/sub), using Ably terms, is a format where publishers send messages to a channel with no knowledge of who the subscribers are. All subscribers on a channel receive the messages with no knowledge of who the publishers are. The ratio of publishers to subscribers can be M:N, across any number of channels.
+
+h2(#publisher). Publisher
+
+A publisher sends messages on a channel.
+
+h2(#push-notifications). Push notifications
+
+Push notifications are messages that are pushed to a device's user interface such as a visual notification. Ably can deliver push notifications using platforms such as Apple's Push Notification service or Google's Firebase Cloud Messaging service.
+
+See the "push notification documentation":/realtime/push for further information.
+
+h2(#queues). Queues
+
+Reactor Queues store messages in the order they were produced and are retrieved by only a single consumer, as opposed to all subscribers when using publish-subscribe. Ably uses integration rules to publish messages received in a channel into a queue.
+
+See the "queues documentation":/general/queues for further information.
+
+h2(#realtime-sdk). Realtime client library SDK
+
+The realtime client library SDKs are used to add persistent realtime connections to Ably within applications.
+
+See the "SDK downloads page":https://ably.com/download for further information on SDKs and the available languages.
+
+h2(#rest-sdk). REST client library SDK
+
+The REST client library SDKs are used to add synchronous interface to Ably within applications.
```

Comment:
The REST client library SDKs provide an API to interact with the Ably REST (HTTP) interface.

Reply:
Fixed.